### PR TITLE
 Add readme-fr.md to french users

### PR DESCRIPTION
**Pull request by Saimyx**
Hey ;)

### CODE REVIEW

**Review by yalh76 — DISMISSED** [review]

File: README_fr.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+**Version incluse :** 5.4
+
+## Captures d'écran
+
+![](https://s.w.org/images/home/screen-themes.png?1)
+
+## Configuration
+
+Utilisez le panneau d'administration de votre wordpress pour le configurer.
```
Comment (ericgaspar) [inline README_fr.md]:
```suggestion
Utilisez le panneau d'administration de votre WordPress pour le configurer.
```

File: README_fr.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+ * Documentation officielle : https://codex.wordpress.org/
+ * Documentation YunoHost : https://yunohost.org/#/app_wordpress
+ 
+## Caractéristiques spécifiques YunoHost
+
+ * Intégration avec les utilisateurs YunoHost et le SSO :
+   * en mode privé : Le blog ou le site est accessible uniquement aux utilisateurs YunoHost
+   * en mode public : Le blog ou le site est accessible par n'importe qui et les utilisateurs YunoHost sont automatiquement connectés
+ * Mises à jour automatiques du cœur de wordpress, extentions et thèmes.
```
Comment (ericgaspar) [inline README_fr.md]:
```suggestion
 * Mises à jour automatiques du cœur de WordPress, extentions et thèmes.
```

File: README_fr.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+#### Architectures supportées
+
+* x86-64b - [![](https://ci-apps.yunohost.org/ci/logs/wordpress%20%28Apps%29.svg)](https://ci-apps.yunohost.org/ci/apps/wordpress/)
+* ARMv8-A - [![](https://ci-apps-arm.yunohost.org/ci/logs/wordpress%20%28Apps%29.svg)](https://ci-apps-arm.yunohost.org/ci/apps/wordpress/)
+* Buster x86-64b - [![](https://ci-buster.nohost.me/ci/logs/wordpress%20%28Apps%29.svg)](https://ci-buster.nohost.me/ci/apps/wordpress/)
+
+## Limitations
+
+* Le multisite n'est disponible que sur des sous-domaines.
+* Comme les mises à jour automatiques ne fonctionnent pas correctement, prennez soin de bien mettre à jour wordpress via le panneau d'administration de wordpress et pas seulement via le panneau d'administration de Yunohost. Pour des raisons de sécurité, vérifiez bien que toutes les mises à jour sont bien installées dans le panneau d'administration de wordpress comme dans le panneau d'administration de Yunohost.
```
Comment (ericgaspar) [inline README_fr.md]:
```suggestion
* Comme les mises à jour automatiques ne fonctionnent pas correctement, prenez soin de bien mettre à jour WordPress via le panneau d'administration de WordPress et pas seulement via le panneau d'administration de YunoHost. Pour des raisons de sécurité, vérifiez bien que toutes les mises à jour sont bien installées dans le panneau d'administration de WordPress comme dans le panneau d'administration de YunoHost.
```

File: README_fr.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+**Sécurité**
+
+Soyez conscients que wordpress est connu pour avoir souvent des risques de sécurité (https://en.wikipedia.org/wiki/WordPress#Vulnerabilities), donc comme c'est le gestionnaire de sites le plus populaire il est la cible des robots et pirates.
```
Comment (ericgaspar) [inline README_fr.md]:
```suggestion
Soyez conscients que WordPress est connu pour avoir souvent des risques de sécurité (https://en.wikipedia.org/wiki/WordPress#Vulnerabilities), donc comme c'est le gestionnaire de sites le plus populaire il est la cible des robots et pirates.
```

File: README_fr.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+Des vulnérabilités peuvent offrir une brêche dans votre wordpress ou dans votre serveur Yunohost (via l'escalade des droits).
```
Comment (ericgaspar) [inline README_fr.md]:
```suggestion
Des vulnérabilités peuvent offrir une brêche dans votre WordPress ou dans votre serveur YunoHost (via l'escalade des droits).
```

File: README_fr.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+
+N'oubliez pas d'appliquer tous les bons principes de sécurité (mots de passe forts, mises à jours fréquentes, ne pas ajouter du code inconnu dans le thème et les extensionts…). En particuler, *gardez votre wordpress à jour le plus possible*.
```
Comment (ericgaspar) [inline README_fr.md]:
```suggestion
N'oubliez pas d'appliquer les principes de sécurité de base (mots de passe forts, mises à jours fréquentes, ne pas ajouter du code inconnu dans le thème et les extensionts…). En particuler, *gardez votre Wordpress à jour le plus possible*.
```

File: README_fr.md [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+
+Par ailleurs, vous pourriez avoir besoin de regarder ce guide :  https://codex.wordpress.org/Hardening_WordPress. Installer des extensions de sécurité peut-être une bonne chose.
+
+## Liens
+
+ * Rapporter un bug : https://github.com/YunoHost-Apps/wordpress_ynh/issues
+ * Site de Wordpress : https://wordpress.org/
```
Comment (ericgaspar) [inline README_fr.md]:
```suggestion
 * Site de WordPress : https://wordpress.org/
```

File: README_fr.md [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+ * Dépôt de Wordpress : https://core.trac.wordpress.org/browser
```
Comment (ericgaspar) [inline README_fr.md]:
```suggestion
 * Dépôt de WordPress : https://core.trac.wordpress.org/browser
```

File: README_fr.md [hunk 9]
```diff
@@ -0,0 +1,82 @@
+# Wordpress pour YunoHost
+
+[![Niveau d'intégration](https://dash.yunohost.org/integration/wordpress.svg)](https://dash.yunohost.org/appci/app/wordpress) ![](https://ci-apps.yunohost.org/ci/badges/wordpress.status.svg) ![](https://ci-apps.yunohost.org/ci/badges/wordpress.maintain.svg)  
+[![Installer Wordpress avec YunoHost](https://install-app.yunohost.org/install-with-yunohost.png)](https://install-app.yunohost.org/?app=wordpress)
+
+*[Read this readme in english.](./README.md)* 
+
+> *Ce package vous permet d'installer Wordpress rapidement et simplement sur un serveur YunoHost.
```
Comment (ericgaspar) [inline README_fr.md]:
```suggestion
> *Ce package vous permet d'installer WordPress rapidement et simplement sur un serveur YunoHost.  
```

File: README_fr.md [hunk 10]
```diff
@@ -0,0 +1,82 @@
+# Wordpress pour YunoHost
```
Comment (ericgaspar) [inline README_fr.md]:
```suggestion
# WordPress pour YunoHost
```

File: README_fr.md [hunk 11]
```diff
@@ -0,0 +1,82 @@
...
+
+[![Niveau d'intégration](https://dash.yunohost.org/integration/wordpress.svg)](https://dash.yunohost.org/appci/app/wordpress) ![](https://ci-apps.yunohost.org/ci/badges/wordpress.status.svg) ![](https://ci-apps.yunohost.org/ci/badges/wordpress.maintain.svg)  
+[![Installer Wordpress avec YunoHost](https://install-app.yunohost.org/install-with-yunohost.png)](https://install-app.yunohost.org/?app=wordpress)
```
Comment (ericgaspar) [inline README_fr.md]:
```suggestion
[![Installer WordPress avec YunoHost](https://install-app.yunohost.org/install-with-yunohost.png)](https://install-app.yunohost.org/?app=wordpress)
```

File: README.md [hunk 12]
```diff
@@ -3,6 +3,8 @@
 [![Integration level](https://dash.yunohost.org/integration/wordpress.svg)](https://dash.yunohost.org/appci/app/wordpress) ![](https://ci-apps.yunohost.org/ci/badges/wordpress.status.svg) [![](https://ci-apps.yunohost.org/ci/badges/wordpress.maintain.svg)](https://github.com/YunoHost/Apps/#what-to-do-if-i-cant-maintain-my-app-anymore-)  
 [![Install Wordpress with YunoHost](https://install-app.yunohost.org/install-with-yunohost.png)](https://install-app.yunohost.org/?app=wordpress)
 
+*[Lire ce readme en français.](./README_fr.md)*
+
 > *This package allow you to install wordpress quickly and simply on a YunoHost server.
```
Comment (ericgaspar) [inline README.md]:
```suggestion
> *This package allow you to install WordPress quickly and simply on a YunoHost server.  
```

File: README.md [hunk 13]
```diff
@@ -3,6 +3,8 @@
 [![Integration level](https://dash.yunohost.org/integration/wordpress.svg)](https://dash.yunohost.org/appci/app/wordpress) ![](https://ci-apps.yunohost.org/ci/badges/wordpress.status.svg) [![](https://ci-apps.yunohost.org/ci/badges/wordpress.maintain.svg)](https://github.com/YunoHost/Apps/#what-to-do-if-i-cant-maintain-my-app-anymore-)  
 [![Install Wordpress with YunoHost](https://install-app.yunohost.org/install-with-yunohost.png)](https://install-app.yunohost.org/?app=wordpress)
```
Comment (ericgaspar) [inline README.md]:
```suggestion
[![Install WordPress with YunoHost](https://install-app.yunohost.org/install-with-yunohost.png)](https://install-app.yunohost.org/?app=wordpress)
```

**Review by ericgaspar — CHANGES_REQUESTED** [review]
WordPress naming with cap

**Review by frju365 — DISMISSED** [review]
LGTM + Code Review

File: README_fr.md [hunk 14]
```diff
@@ -0,0 +1,82 @@
...
+
+**Sécurité**
+
+Soyez conscients que WordPress est connu pour avoir souvent des risques de sécurité (https://en.wikipedia.org/wiki/WordPress#Vulnerabilities), donc comme c'est le gestionnaire de sites le plus populaire il est la cible des robots et pirates.
+Des vulnérabilités peuvent offrir une brêche dans votre WordPress ou dans votre serveur YunoHost (via l'escalade des droits).
+
+N'oubliez pas d'appliquer les principes de sécurité de base (mots de passe forts, mises à jours fréquentes, ne pas ajouter du code inconnu dans le thème et les extensionts…). En particuler, *gardez votre Wordpress à jour le plus possible*.
+
+Par ailleurs, vous pourriez avoir besoin de regarder ce guide :  https://codex.wordpress.org/Hardening_WordPress. Installer des extensions de sécurité peut-être une bonne chose.
```
Comment (kay0u) [inline README_fr.md]:
```suggestion
Par ailleurs, vous pourriez avoir besoin de regarder [ce guide](https://wordpress.org/support/article/hardening-wordpress/). Installer des extensions de sécurité peut-être une bonne chose.
```

**Review by kay0u — APPROVED** [review]

**Pull request merged**